api/envoy/api/v2/eds.proto

```diff
@@ -50,12 +51,35 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   message Policy {
-    // Percentage of traffic (0-100) that should be dropped. This
-    // action allows protection of upstream hosts should they unable to
-    // recover from an outage or should they be unable to autoscale and hence
-    // overall incoming traffic volume need to be trimmed to protect them.
-    // [#v2-api-diff: This is known as maintenance mode in v1.]
-    double drop_overload = 1 [(validate.rules).double = {gte: 0, lte: 100}];
+    reserved 1;
+
+    message DropOverload {
+      // Identifier for the policy specifying the drop.
+      string category = 1 [(validate.rules).string.min_bytes = 1];
+
+      // Percentage of traffic that should be dropped for the category.
+      envoy.type.Percent drop_percentage = 2;
```

Comment:
@vishalpowar I'm a little late here, but I would recommend using `FractionalPercent` here. It will be higher performance in the data path. cc @zuercher @htuch

Reply:
I can do that in another PR (this one is already merged) unless there is any restriction on making such changes in two different PRs.

Reply:
Yeah, sure, `FractionalPercent` makes sense here.